### DISCUSSION TITLE
0.18.0-beta

### DISCUSSION THREAD

**Discussion by redphx** [redphx/better-xcloud-android#128] [Beta]
<img src="https://github.com/user-attachments/assets/7b50edc9-346d-453d-9da1-d1502f2238f3" width="600">

### Added
- Add `Optimize WebView performance` feature.
    - Default is Off
    - This will automatically apply all the neccessary flags in [this guide](https://better-xcloud.github.io/guide/android-webview-tweaks/). If you previously enabled/disabled these flags, you can set them back to `Default`.
    - It's recommended to use with Thorium WebView for the best performance. [Guide](https://better-xcloud.github.io/guide/android-webview-implementation/).

### Changed
- Enable `Force low-latency audio` by default
- Update the UI of WebView selection dialog
- Re-enable the "Log in remotely" feature

### Fixed
- Fix compatibiliy issues with Fire TV Sticks

### What to do if you found a bug
- Report on [Telegram channel](https://t.me/betterxcloudchat) or in [this discusstion] with your device info, WebView info, screenshot/video
- You can re-install the stable version if you want without having to uninstall the beta version

<hr /><em>This discussion was created from the release <a href='https://github.com/redphx/better-xcloud-android/releases/tag/a0.18.0-beta'>0.18.0-beta</a>.</em>

**Comment by KingNothing81** [comment]
Beside "WebViewSurfaceControl" there is also "WebViewSurfaceControlForTV". 
If not enabled, I don't know if this optimization will have an effect on Android TV devices.

**Reply by redphx** [reply]
This flag is only available on Chromium 126+. I handled this case.

**Comment by KingNothing81** [comment]
I think I found two little issues in the current beta version:

When updating the script or changing some settings in the app settings, the toast message "restarting webpage" shows up, but the actual refresh doesn't happen.

And another thing: When giving Thorium WebView a try, the "Better xCloud" labeled button that shows up in the top right corner when you are still logged out, didn't appear for me. So I had to login first to be able to reach the settings again. With the default Android System WebView the button did show up.

**Reply by redphx** [reply]
Both bugs should be fixed in 0.18.0-beta-2 & beta script
